setup.py

```diff
@@ -139,9 +136,9 @@ def build_extension(self, ext: CMakeExtension):
     "ninja",
     "wheel",
     "cmake",
-    "numpy",
-    "pennylane-lightning>=0.22",
-    "pennylane>=0.22",
+    "numpy<1.24",
+    "pennylane-lightning<=0.28",
+    "pennylane<=0.28",
```

Comment:
Reminder to come back later.

Reply:
Fixed in this [PR](https://github.com/PennyLaneAI/pennylane-lightning-kokkos/pull/31).